src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -49,26 +49,48 @@ public static DeclarationModifiers GetSymbolModifiers(this ISymbol symbol)
             this ISymbol symbol,
             bool hideAdvancedMembers,
             Compilation compilation,
-            IMethodSymbol? editorBrowsableAttributeConstructor = null,
-            List<IMethodSymbol>? typeLibTypeAttributeConstructors = null,
-            List<IMethodSymbol>? typeLibFuncAttributeConstructors = null,
-            List<IMethodSymbol>? typeLibVarAttributeConstructors = null,
-            INamedTypeSymbol? hideModuleNameAttribute = null)
+            Optional<IMethodSymbol?> editorBrowsableAttributeConstructor = default,
```

Comment:
optional+nullable doesn't seem necessary.

Reply:
This is to avoid trying to get the symbol repeatedly when we passed in null to indicate it's not in the compilation. See
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Shared/Extensions/ISymbolExtensions.cs,141